modules/manage/pages/topic-iceberg-integration.adoc

```diff
@@ -277,10 +277,12 @@ You'll be able to use to the catalog to load, query, or refresh the Iceberg data
 
 For example, if you have Redpanda cluster configuration properties set to connect to a REST catalog named `demo`:
 
+```
```

Comment:
always try to put the context for the code block. This looks like yaml config, right?